src/main/java/com/everis/utils/CarMessageListener.java

```diff
@@ -0,0 +1,44 @@
+package com.everis.utils;
```

Comment:
I would suggest to move this class to a more meaningful package, this is not really an utility class it contains some business related logic. Maybe something under the control package ?